src/_InputLabel.tsx

```diff
@@ -21,6 +21,7 @@ const InputLabel: React.FC<Props & SxProp> = ({children, disabled, required, vis
         display: 'block',
         color: disabled ? 'fg.muted' : 'fg.default',
         cursor: disabled ? 'default' : 'pointer',
+        alignSelf: 'flex-start',
```

Comment:
Not relevant to this PR. I just noticed the label was clickable even if the user's cursor wasn't on the text.